src/test/regress/expected/seclabel.out

```diff
@@ -0,0 +1,172 @@
+  node_type  |                      result
+---------------------------------------------------------------------
+ coordinator | {"label": "citus classified", "objtype": "table"}
+ worker_1    |
+(2 rows)
+
+SELECT node_type, result FROM get_citus_tests_label_provider_labels('notice(text)') ORDER BY node_type;
+  node_type  |                         result
+---------------------------------------------------------------------
+ coordinator | {"label": "citus unclassified", "objtype": "function"}
+ worker_1    |
+(2 rows)
+
+SELECT node_type, result FROM get_citus_tests_label_provider_labels('v_dist') ORDER BY node_type;
+  node_type  |                      result
+---------------------------------------------------------------------
+ coordinator | {"label": "citus classified", "objtype": "view"}
+ worker_1    |
+(2 rows)
+
+\c - - - :worker_1_port
+SECURITY LABEL ON TABLE a IS 'citus classified';
+SECURITY LABEL ON FUNCTION notice IS 'citus unclassified';
+SECURITY LABEL ON VIEW v_dist IS 'citus classified';
+\c - - - :master_port
+SELECT node_type, result FROM get_citus_tests_label_provider_labels('a') ORDER BY node_type;
+  node_type  |                      result
+---------------------------------------------------------------------
+ coordinator | {"label": "citus classified", "objtype": "table"}
+ worker_1    | {"label": "citus classified", "objtype": "table"}
+(2 rows)
+
+SELECT node_type, result FROM get_citus_tests_label_provider_labels('notice(text)') ORDER BY node_type;
+  node_type  |                         result
+---------------------------------------------------------------------
+ coordinator | {"label": "citus unclassified", "objtype": "function"}
+ worker_1    | {"label": "citus unclassified", "objtype": "function"}
+(2 rows)
+
+SELECT node_type, result FROM get_citus_tests_label_provider_labels('v_dist') ORDER BY node_type;
+  node_type  |                      result
+---------------------------------------------------------------------
+ coordinator | {"label": "citus classified", "objtype": "view"}
+ worker_1    | {"label": "citus classified", "objtype": "view"}
+(2 rows)
+
+DROP TABLE a CASCADE;
+NOTICE:  drop cascades to view v_dist
+DROP FUNCTION notice;
+-- test that SECURITY LABEL statement is actually propagated for ROLES
+SET citus.log_remote_commands TO on;
+SET citus.grep_remote_commands = '%SECURITY LABEL%';
+SECURITY LABEL for citus_tests_label_provider ON ROLE user1 IS 'citus classified';
+NOTICE:  issuing SECURITY LABEL FOR citus_tests_label_provider ON ROLE user1 IS 'citus classified'
+DETAIL:  on server postgres@localhost:xxxxx connectionId: xxxxxxx
+SECURITY LABEL for citus_tests_label_provider ON ROLE user1 IS NULL;
+NOTICE:  issuing SECURITY LABEL FOR citus_tests_label_provider ON ROLE user1 IS NULL
+DETAIL:  on server postgres@localhost:xxxxx connectionId: xxxxxxx
+SECURITY LABEL for citus_tests_label_provider ON ROLE user1 IS 'citus unclassified';
```

Comment:
Should we also add a test where we don't provide the "provider" at all?

```sql
SECURITY LABEL ON ROLE brand_new_role IS 'citus unclassified';
```

From the docs:
> The name of the provider with which this label is to be associated. The named provider must be loaded and must consent to the proposed labeling operation. If exactly one provider is loaded, the provider name may be omitted for brevity.

Reply:
I am skipping this test since I am loading two providers to test the character escaping thing.

Reply:
Can't we reduce the number of providers we have to one; such that we only have the one that requires proper quoting?

Reply:
Hmm okay, then we will not be testing the general case with name that doesn't need quoting, therefore I didn't want to do that. But it's okay I guess.